### DISCUSSION TITLE
Não Tá Fácil Ser Vampiro

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#441] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/nao-ta-facil-ser-vampiro/

**Comment by brunoaugusto** [comment]
Concordo em gênero, número e degrau com a *tagline* do post

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/nao-ta-facil-ser-vampiro/#comment_1160620178>):

> Concordo em gênero, número e degrau com a *tagline* do post

É... Quando eu fiquei sabendo que ele seria o próximo Bátima eu fiquei "Ah, não, véi" e acabou que eu tava certo

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/nao-ta-facil-ser-vampiro/#comment_1160620178>):

> Concordo em gênero, número e degrau com a *tagline* do post

Totalmente excelente! :grin:
